client/.env

```diff
@@ -1,2 +1 @@
-REACT_APP_SERVER_URL=http://localhost:3000
-REACT_APP_API_URL=https://ll.thespacedevs.com/2.2.0/launch/upcoming/?limit=100&format=json&offset=0
\ No newline at end of file
+REACT_APP_SERVER_URL=http://localhost:3000
```

Comment:
Add `.env` in `.gitignore` and remove it from github.

Reply:
This will be taken care in the very next commit for sure.